docs/dev/README.md

```diff
@@ -11,8 +11,7 @@ Note, that the system is still in active development, so:
 
 :::
 
-**zkSync 2.0** is a [ZK rollup](./rollups.md): a trustless protocol that utilises zero-knowledge proofs to provide scalable low-cost transactions on Ethereum. All assets are stored in a single smart
-contract on the mainchain, while computation and storing data are performed off-chain.
+**zkSync 2.0** is a [ZK rollup](./rollups.md): a trustless protocol that utilises zero-knowledge proofs to provide scalable low-cost transactions on Ethereum. All assets are stored in a single smart contract on the mainchain, while computation and storing data are performed off-chain. As all transactions are proven on Ethereum mainchain, users enjoy the same security level as in Ethereum.
```

Comment:
```suggestion
**zkSync 2.0** is a [ZK rollup](./rollups.md): a trustless protocol that utilises zero-knowledge proofs to provide scalable low-cost transactions on Ethereum. All assets are stored in a single smart contract on the mainchain, while computation and storing data are performed off-chain. As all transactions are proven on Ethereum mainchain, users enjoy the same security level as on Ethereum.
```

Reply:
I don't know which of in/on is correct, but if your comment is for being consistent with the first part of the sentence, note that over there it is "on" is for _on (the) mainchain_

Reply:
Yes, that's why I am suggesting `on`

Reply:
In that case, I'll keep it as it is